frontend/packages/topology/src/components/quick-search/QuickSearchDetails.tsx

```diff
@@ -41,6 +42,18 @@ const QuickSearchDetails: React.FC<QuickSearchDetailsProps> = ({ selectedItem })
           {selectedItem.description}
         </Text>
       </TextContent>
+      {selectedItem?.data?.prerequisites?.length > 0 && (
+        <TextContent>
+          <Text component={TextVariants.h6} className="odc-quick-search-details__description">
+            Prerequisites
+          </Text>
+          {selectedItem.data.prerequisites.map((pq: string) => (
+            <Text component={TextVariants.p} className="odc-quick-search-details__description">
+              {pq}
+            </Text>
+          ))}
+        </TextContent>
```

Comment:
This should be part of the quick search catalog item description itself so that the quick search doesn't have special handling for different item types.